kolibri/core/discovery/upgrade.py

```diff
@@ -18,20 +20,22 @@ def move_network_location_entries():
     """
     Move network location entries from the default database to the new standalone database
     """
-    source_queryset = NetworkLocation.objects.using("default").all()
+    if OPTIONS["Database"]["DATABASE_ENGINE"] == "sqlite":
+        # Only do this upgrade on SQLite
+        source_queryset = NetworkLocation.objects.using("default").all()
```

Comment:
super minor note –

the "[return early](https://szymonkrajewski.pl/why-should-you-return-early/)" strategy would decrease nesting and make the diff smaller

```suggestion
    # Only run upgrade on SQLite
    if OPTIONS["Database"]["DATABASE_ENGINE"] != "sqlite":
        return

    source_queryset = NetworkLocation.objects.using("default").all()
```

Reply:
Since the function is very specific to this upgrade step, the return strategy sounds clean to me too, yeah

Reply:
Updated with a force push to reduce the diff.